Reference/Security/two-factor-authentication/index.md

```diff
@@ -0,0 +1,189 @@
+---
+versionFrom: 9.3.0
+keywords: 2fa, security, members
+meta.Title: "Two-factor authentication"
+meta.Description: "Umbraco users and members supports a two-factor authentication (2FA) abstraction for implementing a 2FA provider of your choice"
+---
+
+# Two-factor authentication for Members
+Two-factor authentication (2FA) for Umbraco members are activated by implementing an `ITwoFactorProvider` interface and register the implementation. The implementation can use third-party packages to archive for example support for authentication apps like Microsoft- or Google Authentication App.
+
+Because Umbraco do not control how the UI is for member login and profile edit, the UI for 2FA is shipped at part of the snippets for macros. These can be used as a starting point, before styling the page like you would like.
```

Comment:
```suggestion
Because Umbraco does not control how the UI is for member login and profile edit, the UI for 2FA is shipped as part of the snippets for macros. These can be used as a starting point, before styling the page as you would like.
```